modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/CorrelationInfoTestCase.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.module.extension;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.extension.api.runtime.parameter.CorrelationInfo;
+
+import org.junit.Test;
+
+public class CorrelationInfoTestCase extends AbstractExtensionFunctionalTestCase {
```

Comment:
missing test for OutboundCorrelationStrategy?

Reply:
i think not